src/main/java/com/apifest/oauth20/tests/ClientCredentialsTokenFlowTest.java

```diff
@@ -54,7 +54,7 @@ public void when_invalid_client_id_and_client_secret_in_Authorization_Header_ret
         String accessToken = obtainClientCredentialsAccessToken("invalid_clientId", clientSecret, DEFAULT_SCOPE, true);
 
         // THEN
-        assertTrue(accessToken.contains("{\"error\": \"invalid client_id\"}"));
```

Comment:
These changes are valid but they are already fixed by pull request https://github.com/apifest/apifest-oauth20-tests/pull/2 - merged several minutes ago. I'm sorry for the delay.